apps/web/pages/event-types/[type].tsx

```diff
@@ -1312,6 +1338,109 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                         />
 
+                        <div className="block items-start sm:flex">
+                          <div className="min-w-48 mb-4 sm:mb-0">
+                            <label
+                              htmlFor="recurringEvent"
+                              className="flex text-sm font-medium text-neutral-700">
+                              {t("recurring_event")}
+                            </label>
+                          </div>
+                          <div className="w-full">
+                            <div className="relative flex items-start">
+                              <div className="flex h-5 items-center">
+                                <input
+                                  onChange={(event) => {
+                                    setRecurringEventDefined(event?.target.checked);
+                                    if (!event?.target.checked) {
+                                      formMethods.setValue("recurringEvent", {});
+                                    } else {
+                                      formMethods.setValue(
+                                        "recurringEvent",
+                                        recurringEventDefined
+                                          ? eventType.recurringEvent
+                                          : {
+                                              interval: 1,
+                                              count: 12,
+                                              freq: RRuleFrequency.WEEKLY,
+                                            }
+                                      );
+                                    }
+                                    eventType.recurringEvent = formMethods.getValues("recurringEvent");
+                                  }}
+                                  type="checkbox"
+                                  className="text-primary-600  h-4 w-4 rounded border-gray-300"
+                                  defaultChecked={recurringEventDefined}
+                                />
+                              </div>
+                              <div className="text-sm ltr:ml-3 rtl:mr-3">
+                                <p className="text-neutral-900">{t("recurring_event_description")}</p>
+                              </div>
+                            </div>
+                            <Collapsible
+                              open={recurringEventDefined}
+                              onOpenChange={() => setRecurringEventDefined(!recurringEventDefined)}>
+                              <CollapsibleContent className="mt-4 text-sm">
+                                <div className="flex items-center">
+                                  <p className="mr-2 text-neutral-900">{t("repeats_every")}</p>
+                                  <input
+                                    type="number"
+                                    min="1"
+                                    max="20"
```

Comment:
This is a hardcoded value, I know! `Interval` refers to the X in `every X weeks` for example. Wasn't sure we needed it to be configurable not too big or small of a value, 20 felt right. @Jaibles @PeerRich thoughts? Same hardcoded value for `count` which is the Y in `Max Y weeks`, the  number of occurrences.

Reply:
agree